app/models/service_template_transformation_plan.rb

```diff
@@ -76,6 +76,13 @@ def self.create_catalog_item(options, _auth_user = nil)
   end
 
   def update_catalog_item(options, _auth_user = nil)
+    if config_info[:warm_migration] &&
+       options.key?(:config_info) &&
+       options[:config_info].key?(:warm_migration_cutover_datetime) &&
+       options.dig(:config_info, :warm_migration_cutover_datetime).blank?
```

Comment:
Could this just be `if config_info[:warm_migration] && options.dig(:config_info, :warm_migration_cutover_datetime).blank?` ?  Checking for the existence of the intermediate keys seems like overkill

Reply:
I would agree, but `options.dig(:config_info, :warm_migration_cutover_datetime)` will return `nil` even in situations when none of the two keys tested are set. So we would be deleting the cutover time even in situations when we actually want to update the service template.

In this particular situation, I want to unset the cutover time in a situation when the `options` hash says so, i.e. `options[:config_info][:warm_migration_cutover_datetime] => nil ` or `""`.

Though if there's a better way to achieve this, I'm certainly not opposed to it :smile:

Reply:
@agrare ping -- any take on the above? Thanks.

Reply:
That's a good point if you're trying to minimize removing the cutover time